pkg/api/types.go

```diff
@@ -754,6 +754,8 @@ type EndpointsList struct {
 type NodeSpec struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Cidr represents the IP range assigned to the node
+	Cidr string `json:"cidr,omitempty"`
```

Comment:
PodCIDR sounds fine to me, also. We have multiple address pools (e.g., services, external IPs), so we need to be fairly specific here.